docs/source/api/cachegroups.rst

```diff
@@ -123,19 +130,25 @@ Creates a :term:`Cache Group`
 
 Request Structure
 -----------------
-:fallbackToClosest: If ``true``, the Traffic Router will fall back on the 'closest' :term:`Cache Group` to this one, when this one fails
+:fallbacks: An optional field which, when present, should contain an array of names of other :term:`Cache Groups` on which the Traffic Router will fall back in the event that this :term:`Cache Group` fails/becomes unavailable\ [#fallbacks]_
+
+	.. versionadded:: ATCv4.0
```

Comment:
should this be API version? same comment in the other places that reference this version

Reply:
Unfortunately no. The `fallbacks` array was added to the cachegroup response object in all API versions in ATCv4.

FWIW this is a big part of why I don't think the API ought to be versioned separately from TO.

Reply:
💯 agree very confusing to end users